src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -47,7 +47,13 @@ def handle_exception(ex):
             logger.error(ex)
         return 1
 
+    # Otherwise, unhandled exception. Direct users to create an issue.
+    logger.error("The command failed with an unexpected error. Here is the traceback:\n")
     logger.exception(ex)
+    logger.error("\nIf you think this is a bug, please visit: https://github.com/Azure/azure-cli to create an issue."
```

Comment:
To make the "If you think this is... blah" more visually distinct, I would suggest using `logger.warning` here.

Reply:
Will update.